code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/LdapConfig.java

```diff
@@ -0,0 +1,153 @@
+package io.cattle.platform.iaas.api.auth.integration.ldap;
+
+import io.cattle.platform.iaas.api.auth.integration.interfaces.AuthConfig;
+import io.github.ibuildthecloud.gdapi.annotation.Field;
+import io.github.ibuildthecloud.gdapi.annotation.Type;
+
+@Type(name = LdapConstants.CONFIG)
```

Comment:
The api for this ends up being "ldapconfig". I realize that that is consistent with "githubconfig" but everything else in the api is camelCase. Can you make this camelcase without breaking the world?
...no need to refactor githubconfig bc that would probably cause problems for vince. I think.